src/OpenTelemetry.AutoInstrumentation.StartupHook/StartupHook.cs

```diff
@@ -123,7 +123,10 @@ private static List<string> GetExcludedApplicationNames()
             return excludedProcesses;
         }
 
-        foreach (var processName in environmentValue.Split(','))
+        // The separator should always match the Settings.Separator. However, to avoid
```

Comment:
There is a construction for such things (Shared projects) https://stackoverflow.com/questions/24294608/how-do-i-create-a-shared-code-project-shproj
but in this case it will be probably overengineering.